README.md

```diff
@@ -1 +1,241 @@
-# pino-test
\ No newline at end of file
+# pino-test
+
+A set of API to easily test the Pino logger.
+
+## Getting started
+```sh
+npm install pino-test --save-dev
+```
+
+```js
+const test = require('node:test')
+const pino = require('pino')
+const pinoTest = require('pino-test')
+
+test('pino should log a info message', async () => {
+  const stream = pinoTest.sink()
+  const logger = pino(stream)
+
+  logger.info('hello world')
+
+  const expected = { msg: 'hello world', level: 30 }
+  await pinoTest.once(stream, expected)
+})
+```
+
+Using tap
+```js
+const pino = require('pino')
+const pinoTest = require('pino-test')
+
+test('pino should log a info message on tap', async () => {
+  const stream = pinoTest.sink()
+  const logger = pino(stream)
+
+  logger.info('hello world')
+
+  const expected = { msg: 'hello world', level: 30 }
+  await pinoTest.once(stream, expected)
+})
+
+// with a own assert function
+test('pino should log a info message using a own assert function', async ({ same }) => {
+  const stream = pinoTest.sink()
+  const logger = pino(stream)
+
+  logger.info('hello world')
+
+  const expected = { msg: 'hello world', level: 30 }
+  await pinoTest.once(stream, expected, same)
+})
+```
```

Comment:
The only difference between this example and the `node:test` example is the including of a second test that utilizes a custom assertion function. I think we can move that example into the `node:test` block and completely remove the `tap` example as it is redundant.